### ISSUE TITLE
Using buffering on socket ports spews some debug log into the current-output-port

### ISSUE THREAD

**Issue by kdltr** [ursetto/socket-egg#2]
Hi!

Some debug print statements are still in the code, and are reached when using buffering with, for example, the tcp6 egg, and the size of the buffer is not a multiple of the length of the written data.

This can flood the output with the content of the buffer, and worse: if current-output-port is the socket port itself, it will enter an infinite recursion, spewing the buffer infinitely until memory is exhausted, making the program crash.

Here is a simple program that exposes the behaviour:

```
(import tcp6)
(tcp-buffer-size 256)

(define-values (in out) (tcp-connect "127.0.0.1" 2345))

(let lp ()
  (display "AAA" out)
  (lp))
```

Sample output of the above program:
```
(slop 2 end 256)
(s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA)
(slopping)
```

**Comment by ursetto** [comment]
Fixed by #3

**Comment by kdltr** [comment]
Thanks! Don’t forget to do a new release so it hits the eggs caches! ^^
